peekingduck/pipeline/nodes/model/hrnetv1/hrnet_files/detector.py

```diff
@@ -0,0 +1,158 @@
+"""
+Copyright 2021 AI Singapore
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import logging
+from typing import Any, Dict, List, Tuple, Callable
+
+import tensorflow as tf
+import numpy as np
+from peekingduck.utils.graph_functions import load_graph
+from peekingduck.pipeline.nodes.model.hrnetv1.hrnet_files.preprocessing \
+    import project_bbox, box2cs, crop_and_resize
+from peekingduck.pipeline.nodes.model.hrnetv1.hrnet_files.postprocessing \
+    import scale_transform, affine_transform_xy, reshape_heatmaps, \
+    get_valid_keypoints, get_keypoint_conns, get_bbox_of_poses
```

Comment:
Can remove `get_bbox_of_poses` since we not going to report bbox

Reply:
okay, removed